packages/spec/integration/graphql/mixin.runtime.js

```diff
@@ -0,0 +1,23 @@
+const { Mixin } = require('hops-mixin');
+const fetch = require('cross-fetch');
+const { HttpLink } = require('apollo-link-http');
+
+const customFetch = getServerAddress => async (uri, options) => {
+  const urlSuffix =
+    typeof Headers !== 'undefined' && options.headers instanceof Headers
+      ? options.headers.get('x-url-suffix')
+      : options.headers['x-url-suffix'];
+  const newUri = urlSuffix ? `${getServerAddress()}/graphql/${urlSuffix}` : uri;
+
+  return await fetch(newUri, options);
+};
+
+module.exports = class GraphQlMixin extends Mixin {
+  getApolloLink() {
+    return new HttpLink({
+      uri: this.config.graphqlUri,
+      // TODO: get correct server address
+      fetch: customFetch(() => this.serverAddress || 'http://localhost:8080'),
```

Comment:
`this.getServerAddress()` should give you that (https://github.com/untool/untool/blob/master/packages/express/mixins/mixin.core.js#L38)

Reply:
Leads to `Error: this.getServerAddress is not a function` for me. And I don't really have an idea how to get this information asynchronously from within the core-mixin… :-|

Reply:
How about `res.locals.serverAddress = this.getServerAddress()`?